src/views/admin/users/page.tsx

```diff
@@ -74,6 +75,7 @@ const userRow = (user: UserType) => {
         <Tag key={i}>{role}</Tag>
       ))}
     </div>,
+    <Button>Delete</Button>
```

Comment:
use appropriate icons instead of text in the buttons.
Add a button to disable the account. also add appropriate aria attributes for accessibility.

Reply:
Is there any API u included by using which i can access icons ?